reporter-config3/src/main/java/com/addthis/metrics3/reporter/config/MetricFilterTransformer.java

```diff
@@ -15,63 +15,72 @@
 package com.addthis.metrics3.reporter.config;
 
 import com.addthis.metrics.reporter.config.PredicateConfig;
+
+import com.codahale.metrics.Histogram;
+import com.codahale.metrics.Meter;
 import com.codahale.metrics.Metric;
 import com.codahale.metrics.MetricFilter;
+import com.codahale.metrics.Timer;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class MetricFilterTransformer
-{
+import static com.addthis.metrics.reporter.config.PredicateConfig.allowMeasurement;
+
+public class MetricFilterTransformer {
 
     private static final Logger log = LoggerFactory.getLogger(MetricFilterTransformer.class);
 
-    private static class PredicateConfigFilter implements MetricFilter
-    {
+    private static class PredicateConfigFilter implements MetricFilter {
         final PredicateConfig predicate;
 
-        PredicateConfigFilter(PredicateConfig predicate)
-        {
+        PredicateConfigFilter(PredicateConfig predicate) {
             this.predicate = predicate;
         }
 
         @Override
-        public boolean matches(String name, Metric metric)
-        {
-            log.trace("Checking Metric name: {} {}", new Object[] {name, unqualifyMetricName(name)});
-            if (predicate.getUseQualifiedName())
-            {
-                return predicate.allowString(name);
+        public boolean matches(String name, Metric metric) {
+            log.trace("Checking Metric name: {} {}", new Object[]{name, unqualifyMetricName(name)});
+
+            boolean allowString;
+            if (predicate.getUseQualifiedName()) {
+                allowString = predicate.allowString(name);
+            } else {
+                allowString = predicate.allowString(unqualifyMetricName(name));
+            }
+
+            if (!allowString) {
```

Comment:
I think I'd prefer this being if (allowString) { // do stuff and change allowString }; return allowString. Makes it more readable imo